splunk-on-ubuntu/azuredeploy.json

```diff
@@ -381,6 +420,18 @@
           "contentVersion": "1.0.0.0"
         },
         "parameters": {
+          "SplunkURL": {
```

Comment:
It's not. In your system of templates, once you add a parameter one place, you have to add it almost everywhere else so the template substitutions work.